domain/src/main/java/org/oppia/android/domain/topic/TopicController.kt

```diff
@@ -473,6 +474,7 @@ class TopicController @Inject constructor(
           putAllWrittenTranslations(topicRecord.writtenTranslationsMap)
           title = topicRecord.translatableTitle
           description = topicRecord.translatableDescription
+          classroomId = topicRecord.classroomId
```

Comment:
As mentioned in another comment, I'd instead expect that we'd load a topic in the context of a classroom (that is, pass the classroom ID down into ``TopicController``).

Reply:
This is now removed.